syncer/syncer.go

```diff
@@ -1753,6 +1759,12 @@ func (s *Syncer) genRouter() error {
 func (s *Syncer) printStatus(ctx context.Context) {
 	defer s.wg.Done()
 
+	failpoint.Inject("PrintStatusCheckSeconds", func(val failpoint.Value) {
+		seconds, _ := val.(int)
```

Comment:
The `val` will be `struct{}{}` if the failpoint expression is `return` which without any value and seconds will be 0. Two ways to fix it:

- type assert directly and make it panic if failpoint expression is unexpected.
- set a default value for statusTime if `!ok.`